engine/src/CalciteExpressionParsing.cpp

```diff
@@ -258,42 +251,19 @@ cudf::type_id get_output_type_expression(const cudf::table_view & table, std::st
 			interops::operator_type operation = map_to_operator_type(token);
 			if(is_binary_operator(operation)) {
 				if(operands.size() < 2)
-					throw std::runtime_error(
-						"In function get_output_type_expression, the operator cannot be processed on less than one or "
-						"zero elements");
+					RAL_FAIL("In function get_output_type_expression, the operator cannot be processed on less than one or zero elements");
 
 				cudf::type_id left_operand = operands.top();
 				operands.pop();
 				cudf::type_id right_operand = operands.top();
 				operands.pop();
 
-				// TODO percy cudf0.12 was invalid here, should we consider empty?
```

Comment:
Porque se borro esto? Si los types EMPTY nunca deberian de suceder seria bueno, por lo menos que obtengamos un error predecible cuando eso suceda

Reply:
Ahh creo que entiendo, esta verificacion la estas haciendo dentro de otra funcion

Reply:
yeah that check now is done in `get_output_type`